pyogrio/tests/test_geopandas_io.py

```diff
@@ -1658,11 +1682,12 @@ def test_write_read_mixed_column_values(tmp_path):
     write_dataframe(test_gdf, output_path)
     output_gdf = read_dataframe(output_path)
     assert len(test_gdf) == len(output_gdf)
-    for idx, value in enumerate(mixed_values):
-        if value in (None, np.nan):
-            assert output_gdf["mixed"][idx] is None
-        else:
-            assert output_gdf["mixed"][idx] == str(value)
+    # mixed values as object dtype are currently written as strings
```

Comment:
```suggestion
    # non-null mixed values as object dtype are currently written as strings
```

Reply:
(also null values are written as string _dtype_, but preserving the fact they are null)